content/en/security_platform/application_security/getting_started/php.md

```diff
@@ -0,0 +1,84 @@
+---
+title: PHP Applications
+kind: documentation
+code_lang: php
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-php'
+      tag: 'GitHub'
+      text: 'PHP Datadog Library source code'
+---
+
+{{% appsec-getstarted %}}
+
+## Get started
+
+1. **Install the latest Datadog PHP library** by downloading and running the installer:
+   ```
+   wget https://raw.githubusercontent.com/DataDog/dd-appsec-php/installer/dd-library-php-setup.php
+   php dd-library-php-setup.php --php-bin all --tracer-version latest --appsec-version latest
+   ```
+
+2. **Enable the library in your code** by restarting PHP-FPM or Apache. In a containerized environment, use the following configuation to enable Application Security:
```

Comment:
The library will already be enabled after installation (when using `--appsec-*` on the installer) so the container specific environment settings should be mentioned as an optional step.